Chapter07/recipe-04/c-example/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
+
+project(recipe-04 C)
+
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_EXTENSIONS OFF)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+
+include(GNUInstallDirs)
+include(ExternalProject)
+
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  set(CMAKE_INSTALL_PREFIX "/usr/local/${PROJECT_NAME}" CACHE PATH "Install path" FORCE)
```

Comment:
why is this needed? ignore my question if the book will give more context.

Reply:
To make sure it's set to something sensible. This is suggested in the CMake docs.

Reply:
OK thanks! We need to figure out how these things behave on Windows at some point :-)

Reply:
😱